mycore-media/src/main/java/org/mycore/media/services/MCRPdfThumbnailGenerator.java

```diff
@@ -43,4 +49,30 @@ public boolean matchesFileType(String mimeType, MCRPath path) {
             return Optional.of(pdfRender);
         }
     }
+
+    private PDPage resolveFirstPage(PDDocument pdf) throws IOException {
```

Comment:
better name: resolveInitialPage
first should be used for listable items and should return the head of the list. In this case the returned page can actually be the 7th